go.mod

```diff
@@ -84,6 +84,7 @@ replace (
 	k8s.io/component-base => k8s.io/component-base v0.0.0-20191016111319-039242c015a9
 	k8s.io/cri-api => k8s.io/cri-api v0.0.0-20190828162817-608eb1dad4ac
 	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.0.0-20191016115521-756ffa5af0bd
+	k8s.io/klog => k8s.io/klog v0.4.0
```

Comment:
I wouldn't use `replace` for klog since we're using a valid tag.

Reply:
Setting `require` to `k8s.io/klog v0.4.0`, `k8s.io/api v0.17.0`, and `k8s.io/apimachinery v0.17.0`, and dropping references to them in the `replace` section, `update-deps.sh` using go 1.12 produces this output:
```
go: finding k8s.io/api v0.0.0
go: finding k8s.io/apimachinery v0.0.0
go: k8s.io/api@v0.0.0: unknown revision v0.0.0
go: k8s.io/apimachinery@v0.0.0: unknown revision v0.0.0
go: error loading module requirements
```